pages/index.mdx

```diff
@@ -0,0 +1,224 @@
+  {
+    label: "Kotlin",
+    icon: kotlinGray,
+    iconActive: kotlinColor,
+    href: "https://github.com/Dwolla/dwolla-v2-kotlin",
+  },
+];
+
+<!-- Partners and Integrations LogoLink props -->
+
+export const partnerContent = [
+  {
+    label: "Plaid",
+    icon: plaidGray,
+    iconActive: plaidColor,
+    href: "https://www.dwolla.com/integrations/plaid/",
+  },
+  {
+    label: "Sift",
+    icon: siftGray,
+    iconActive: siftColor,
+    href: "https://www.dwolla.com/integrations/sift/",
+  },
+  {
+    label: "Slack",
+    icon: slackGray,
+    iconActive: slackColor,
+    href: "https://www.dwolla.com/partnerships/slack/",
+  },
+  {
+    label: "Quickbooks",
+    icon: quickbooksGray,
+    iconActive: quickbooksColor,
+    href: "https://www.dwolla.com/partnerships/quickbooks/",
+  },
+  {
+    label: "Apto",
+    icon: aptoGray,
+    iconActive: aptoColor,
+    href: "https://www.dwolla.com/contact/apto-waitlist/",
+  },
+];
+
+export default function HomePage() {
+  return (
+    <>
+      <BannerWrapper>
+        <Banner
+          topic="Integrate with Dwolla and move money smarter."
+          description="From ideation, to implementation, and beyond, the Dwolla API is your connection to the US banking system."
+          button={{
+            text: "Discover the Possibilites",
+            link: {
+              href: "https://accounts-sandbox.dwolla.com/login",
+              external: true,
+            },
+          }}
+        />
+      </BannerWrapper>
+      <ContentContainer>
+        <CardGrid>
+          <Link href="/guides" key="Guides" passHref>
+            <StyledLink>
+              <Card
+                icon={guideIcon}
+                topic="Guides"
+                description="Explore step-by-step walkthroughs to ensure your application is built to best practices."
+              />
+            </StyledLink>
+          </Link>
+          <Link href="/concepts" key="Concepts" passHref>
+            <StyledLink>
+              <Card
+                icon={conceptIcon}
+                topic="Concepts"
+                description="Transform your use case to business and functional requirements."
+              />
+            </StyledLink>
+          </Link>
+          <StyledLink
+            href="https://docs.dwolla.com/"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <Card
+              icon={apiReferenceIcon}
+              link
+              topic="API Reference"
+              description="View details for all of Dwolla’s API endpoints with a complete reference"
+            />
+          </StyledLink>
+        </CardGrid>
+        <ExternalCTA
+          icon={developerCommunityIcon}
+          title="Developer Support"
+          description="Interact with the developer community and Dwolla developer support to find answers to your technical questions."
+          href="https://discuss.dwolla.com/"
+        />
+        <H3>SDKs &amp; Tools</H3>
```

Comment:
For the h3's on this page, could we add a bit more space above them and remove a bit of the space below them to line up with the designs a bit better? The sections feel like the blend together a bit too much right now

Reply:
Good call! I can add that!